data/model/_basequery.py

```diff
@@ -18,7 +18,6 @@
     Namespace,
     Visibility,
     ImageStorage,
-    Image,
     RepositoryKind,
     db_for_update,
     db_count_estimator,
```

Comment:
I see references to `Image` in this file for example https://github.com/quay/quay/blob/master/data/model/_basequery.py#L186

Reply:
looks like it's being used in the function `set_image_storage_metadata` but that function is never called so we could probably remove the whole flow